frontend/app/routes/public/renew/$id/adult-child/review-adult-information.tsx

```diff
@@ -182,9 +180,7 @@ export async function action({ context: { serviceProvider, session }, params, re
     }
   }
 
-  const submissionInfo = await serviceProvider.getBenefitRenewalService().createBenefitRenewal(state);
-
-  saveRenewState({ params, session, state: { submissionInfo } });
+  saveRenewState({ params, session, state: {} });
 
   return redirect(getPathById('public/renew/$id/adult-child/review-child-information', params));
```

Comment:
I wonder if we should be submitting the renewal here if there are no children being renewed.

I asked the designers in the Figma.